client/me/purchases/manage-purchase/index.tsx

```diff
@@ -327,6 +327,16 @@ class ManagePurchase extends Component<
 		return hasSetupAds && purchase && isPlan( purchase );
 	}
 
+	isPendingDomainRegistration( purchase: Purchase ): boolean {
+		if ( isDomainRegistration( purchase ) ) {
+			const domain = this.props.domainsDetails[ purchase.siteId ].find(
+				( domain ) => domain.name === purchase.meta
+			);
+			return domain?.pendingRegistrationAtRegistry ?? false;
+		}
+		return false;
```

Comment:
```suggestion
		if ( ! isDomainRegistration( purchase ) ) {
			return false;
		}

		const domain = this.props.domainsDetails[ purchase.siteId ].find(
			( domain ) => domain.name === purchase.meta
		);
		return domain?.pendingRegistrationAtRegistry ?? false;
```
Just a small suggestion 🙂 I know this is a tiny method and it doesn't make much difference, but I think early returns are more used in our codebase.

Reply:
Definitely better! Thanks for the suggestion! :)